README.md

```diff
@@ -154,6 +154,21 @@ k3sup app install --help
 k3sup app install APP_NAME --help
 ```
 
+Our currently supported apps:
+* OpenFaaS
+* nginx-ingress
+* cert-manager
+* openfaas-ingress
+* inlets-operator
```

Comment:
Is some of this duplicated in another section of the file?

Reply:
The section just above this list (part of this "section") has some further examples of a couple of the apps, this is just a bullet list of all the commands.